x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -411,7 +412,7 @@ public void testForceMergeAction() throws Exception {
         };
         assertThat(numSegments.get(), greaterThan(1));
 
-        createNewSingletonPolicy("warm", new ForceMergeAction(1));
+        createNewSingletonPolicy("warm", new ForceMergeAction(1, Codec.getDefault()));
```

Comment:
Can you add an integration test for the best_compression codec here? (in a separate test method)

Reply:
Thinking of adding codec as parameterized and use this test for best_compression as well. What you think?